### PR TITLE
ANN: Support path highlight for private types

### PR DESCRIPTION
**Pull request by farodin91**

### CODE REVIEW

**Comment by farodin91** [comment]
#886

File: src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt [hunk 1]
```diff
@@ -111,7 +111,7 @@ abstract class RsFunctionImplMixin : RsStubbedNamedElementImpl<RsFunctionStub>,
 
     constructor(stub: RsFunctionStub, nodeType: IStubElementType<*, *>) : super(stub, nodeType)
 
-    override val isPublic: Boolean get() = RsPsiImplUtil.isPublicNonStubbed(this)
+    override val isPublic: Boolean get() = RsPsiImplUtil.isPublic(this, stub)
```
Comment (vlad20012) [inline src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt]: :+1:

**Comment by vlad20012** [comment]
bors r+
Thanks!

**Comment by vlad20012** [comment]
Hmm, looks like it's a real test failure, @farodin91

**Comment by farodin91** [comment]
@vlad20012 should be fixed

**Comment by farodin91** [comment]
bors try

**Comment by matklad** [comment]
delegate+

@farodin91 now try should work I think...

**Comment by matklad** [comment]
bors delegate+

**Comment by farodin91** [comment]
bors try
test

**Comment by matklad** [comment]
bors try

**Comment by vlad20012** [comment]
We finally tested delegate+! And... it doesn't work.

**Comment by vlad20012** [comment]
bors try

**Comment by vlad20012** [comment]
Seems ok
bors r+

**Pull request merged**